cdap-app-templates/cdap-etl/cdap-etl-archetypes/cdap-etl-batch-source-archetype/src/main/resources/archetype-resources/widgets/Source-batchsource.json

```diff
@@ -0,0 +1,13 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "configuration-groups": [
+    {
+      "label": "Source Configuration",
+      "properties": [
```

Comment:
adding the properties here is very cdap specific, meaning not a common standard, we should add a link for reference here.